### ISSUE TITLE
Redirect after submitting a form

### ISSUE THREAD

**Issue by ottoo** [davidkpiano/react-redux-form#300]
In my forms onSubmit handler function, I am trying to redirect to another route that is rendered in the same container by dispatching an action. Here, pushing or replacing the browserHistory causes an error `Uncaught (in promise) TypeError: Cannot set property 'submit' of null(…)`. 

In the container: 

```
const mapDispatchToProps = dispatch => ({
    doLogin(user) {
        dispatch(actions.submit('login', login(user).then(res => {
            if (res && res.isValid) {
                sessionStorage.setItem('jwtToken', res.token);
                browserHistory.replace('/notes'); // Problem
            }
        })));
    }
});
```

If I remove the route change, form submits fine. Does anybody know whats the issue here?

**Comment by ottoo** [comment]
Originates from form-component 

```
attachNode(node) {
    this._node = node;

    this._node.submit = this.handleSubmit;
  }
```

Looks like when redirecting to another route it cant find the form to attach the submit to or something.. ?

**Comment by i4got10** [comment]
Get this error then updated react from 15.1.0 to 15.2.0

**Comment by davidkpiano** [comment]
This is available in `0.13.11`
